enterprise/tpa_pipeline.py

```diff
@@ -132,9 +138,14 @@ def get_user_social_auth(user, enterprise_customer):
 
     Arguments:
         user (User): user object
-        enterprise_customer (EnterpriseCustomer): User id of user in third party LMS
+        enterprise_customer (EnterpriseCustomer): Instance of the enterprise customer.
 
     """
+    # Give the priority to default IDP of given enterprise when getting social auth entry of user. If found then
```

Comment:
Code repetition ... can we create another local method?

Reply:
This is the same method of calling at both places. I added comments at both places where this is using. I have also added the comment in the function about what it is doing.